docs/manual-test.md

```diff
@@ -97,59 +89,6 @@ Right-click the data source, then click **Extract Data**. Verify that you can cr
 
 For more information, see [Refresh Extracts](https://onlinehelp.tableau.com/current/pro/desktop/en-us/extracting_refresh.htm) in Tableau Desktop and Web Authoring Help.
 
-__Edit your connection__
-
-Change all possible items and verify that changes are applied.
-
-1. Right-click the data source and click **Edit Data Source**.<br/>
-![]({{ site.baseurl }}/assets/mt-edit-data-source.png)
-The worksheet opens in Tableau.
-1. In the left pane, under **Connections**, click the dropdown menu next to the server name and click **Edit Connection**.<br/>
-![]({{ site.baseurl }}/assets/mt-edit-connection.png)
-1. Change something. For example, change the server.<br/>
-![]({{ site.baseurl }}/assets/mt-change-server.png)
-1. After you click **Sign In**, you should see the new server name under **Connections**.<br/>
-![]({{ site.baseurl }}/assets/mt-new-server.png)
```

Comment:
I think this should be kept, edit connection isn't that onerous and I think there are some scenarios where the connector changes can break it.

Reply:
Sounds good. Are there options other than "change the server" we could suggest?

Reply:
reverted

Reply:
Could change the database, or the user account, or auth method. Any of the fields should suffice.